src/sentry/static/sentry/app/views/organizationGroupDetails/eventToolbar.jsx

```diff
@@ -117,13 +113,11 @@ const GroupEventToolbar = createReactClass({
     });
 
     return (
-      <div key="related-transaction">
-        <RelatedTransactionButton title={t('Related Transaction')} to={to} size="small">
-          {t('Related Transaction')}
-        </RelatedTransactionButton>
-      </div>
+      <Button key="related-transaction" to={to} size="small">
```

Comment:
Is `key` needed?

Reply:
Yes, without the key react complained about children in a list.